tests/files/features/.lagoon.yml.test

```diff
@@ -0,0 +1,53 @@
+docker-compose-yaml: docker-compose.yml
```

Comment:
can we call this file something else than `.lagoon.yml.test`? give it a name like `.lagoon-no-cronjobs.yml`

Reply:
Renamed as suggested in 7e4bba47